### PR TITLE
fix: ignore cases on type names in manifest files

### PR DESCRIPTION
**Pull request by mcarvin8**
I was testing this plugin out with my new plugin that combines package.xml files (https://github.com/mcarvin8/sf-package-combiner). This drove me crazy in figuring out why I couldn't get your plugin to work with my plugin until I realized it was casing 😆 

Because Salesforce allows the `<name>` key in a package.xml to be case insensitive (ex: `apexclass`), I updated my plugin to convert all cases to lower-case to ensure multiple package types in multiple packages are always grouped together correctly. 

Ideally, you don't need to worry about this if you use sfdx-git-delta. But if you make manifest files manually or some other way, you can't always ensure the name cases match since Salesforce does not care about the cases of the metadata name. The members are the only items case sensitive since that needs to match API names in Salesforce.

Please accept this fix to ignore cases in just the `<name>` keys in a manifest file.

I'm maintaining the "normalized" case type names that you have hard-coded elsewhere (these line up with how a sfdx-git-delta package would be created as well): 
- ApexClass
- ApexTrigger
- ApexTestSuite

So basically, no matter the case used in the package.xml, it will be assigned to one of the 3 above "normalized" names.

### CODE REVIEW

**Comment by mcarvin8** [comment]
Current test progress. Will update once all tests pass in my fork:

image

**Comment by mcarvin8** [comment]
@renatoliveira - Have you had a chance to look into this request yet? No big deal if you haven't

**Comment by renatoliveira** [comment]
I am so sorry, @mcarvin8. I did not have the time to review this yet. This one passed unnoticed in the middle of many other emails from GitHub about the dependency updates! I'll review this as soon as possible, today.

**Comment by renatoliveira** [comment]
Released 1.9.3 with this change: #114.

**Comment by mcarvin8** [comment]
All good. thank you for accepting this!
